src/main/java/com/java_professional/pnivchuk/homework/lesson_2/Task_1/Main.java

```diff
@@ -0,0 +1,44 @@
+package com.java_professional.pnivchuk.homework.lesson_2.Task_1;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+/*Используя коллекцию удвойте слово: 1. Введите с клавиатуры 5 слов в список строк. 2. Метод doubleValues должен удваивать
+слова по принципу a,b,c -> a,a,b,b,c,c. 3. Используя цикл for выведите результат на экран, каждое значение с новой строки.*/
+
+public class Main {
+    public static void main(String[] args) {
+        List<String> list = new ArrayList<>();
+        List<String> newList = new ArrayList<>();
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+        try {
+            for (int i = 0; i < 5; i++) {
+                System.out.println("Please, input word " + (i+1));
+                list.add(reader.readLine());
+            }
+        }catch(IOException e){
+            System.out.println("O-la-la!!! Something went wrong!");
+            e.getStackTrace();
+        }
+
+        newList = doubleValues(list);
+
+        for(int a = 0; a < newList.size(); a++){
+            System.out.println(newList.get(a));
+        }
+    }
+
+    public static List<String> doubleValues(List<String> list){
+        List<String> newList = new ArrayList<>();
+        for(int i = 0; i < list.size(); i++){
+            for(int j = 0; j < 2; j++){
+                newList.add(list.get(i));
+            }
+        }
+        return newList;
```

Comment:
Than duplicating items in a list, is better to use stringBuilder or concatenation of strings.
For what you need new list? All this operations you can perform in one list.

Reply:
Because i can't change collection size while iterating it.
"is better to use stringBuilder or concatenation of strings" - but i am supposed to return twice as big list, or not? not with the same size but changed items like list[0] = "word" then list[0] = "wordword".

Reply:
Ok, It works, however you should try to avoid quadratic operations.
```suggestion
     public static List<String> doubleValuesOptimized(List<String> list) {
            list.replaceAll(item -> item + ", " + item);
        return list;
    }
```